tests/utils/utils.test.ts

```diff
@@ -0,0 +1,292 @@
+import { normalizeName } from '../../src/client/utils/cardutil';
```

Comment:
I thought I was going to be changing the turnToTree logic so I wrote tests, which are always nice to have regardless.